docs/whats-new/changelog.md

```diff
@@ -2,6 +2,6 @@
 
 
 :::{include} ./changelog.md
-:start-line: 1
+:start-line: 3
```

Comment:
Literally i spent the past 2 weeks trying to understand where this recursion issue was stemming from. I thought it was autodoc so i was making it too complicated. myst-parser actually was spinning on the changelog page because (i'm guessing from knowing its quirks) it doesn't like it when a page has two H1 header tags.

i may report this as a bug to our myst-friends.